.evergreen/install-and-bootstrap-mongosh.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -o errexit  # Exit the script with error if any of the commands fail
+set -o xtrace   # Write all commands first to stderr
+
+source "${PROJECT_DIRECTORY}/.evergreen/init-nvm.sh"
+
+bootstrap_mongosh() {
+	npm cache clear --force || true
+	npm i -g npm@8.x || true
+
+	npm pack | tee npm-pack.log
+
+	npm cache clear --force || true
+	npm i -g npm@8.x || true
+
+	export TARBALL_FILENAME="$(tail -n1 npm-pack.log)"
+	cd /tmp
+	git clone --depth=10 https://github.com/baileympearson/mongosh.git $TASK_ID
+	cd $TASK_ID
+
+	export REPLACE_PACKAGE="mongodb:${PROJECT_DIRECTORY}/${TARBALL_FILENAME}"
+	npm run replace-package
+
+	npm run bootstrap -- --no-ci
+}
+
+bootstrap_mongosh
+
+npm run compile-ts
```

Comment:
Just be aware that this creates a greater dependency on the exact scripts in mongosh and you may need to adjust this more frequently :)